crates/config/src/errors.rs

```diff
@@ -3,6 +3,12 @@
 pub enum ConfigFileError {
     #[error(transparent)]
     IO(#[from] std::io::Error),
+    #[error("no such file or directory: {0}")]
+    NotFound(String),
+    #[error("no such file or directory: {0}")]
+    IncludedFileNotFound(String),
```

Comment:
I think `NotFound` should be `ConfigNotFound` and should have a differentiated error message from the current `IncludedFileNotFound`. Additionally, I'd probably rename `IncludedFileNotFound` to `IncludedConfigNotFound`.

I think these should be the error messages:
1. `"config file not found: {0}"`
2. `"included config file not found: {0}"`